test/unit/iwdp-specs.js

```diff
@@ -0,0 +1,76 @@
+// transpile:mocha
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import IWDP from '../../lib/iwdp';
+import { SubProcess } from 'teen_process';
+import request from 'request-promise';
+import Promise from 'bluebird';
+chai.should();
+chai.use(chaiAsPromised);
+
+const { expect } = chai;
+let iwdpInstance;
+
+describe('ios webkit debug proxy class', async () => {
+  beforeEach(async () => {
+    iwdpInstance = new IWDP();
+  });
+
+  afterEach(async () => {
+    try {
+      await iwdpInstance.stop();
+    } catch (ign) { }
+  });
+
+  it('should detect that IWDP is supported on this machine', function () {
+    expect(IWDP.isSupported());
+  });
+
+  it('should start IWDP and be able to access the main page', async function () {
+    await iwdpInstance.start();
```

Comment:
For this to work you'll need to update `.travis.yml` to install the webkit proxy. The Travis VM has homebrew on it, so it should be pretty easy.